### DISCUSSION TITLE
Several _geo locations

### DISCUSSION THREAD

**Discussion by martin-landhi** [meilisearch/product#681] [Feedback & Feature Proposal]
Is meilisearch Geolocation feature allow have several locations per document?

For example:

I have a companies that have projects in several locations. I want to search for companies that are working on some Location.

Meilisearch allow to create the document for those comapanies with all the locations where they are working and then search?

**Comment by macraig** [comment]
Hi @martin-landhi , thank you for taking the time to explain your use case. We already have an open discussion about this in https://github.com/meilisearch/product/discussions/508, so I'm closing this one in favor of it.
